test/loco/integer/sendmoremoney2.clj

```diff
@@ -0,0 +1,80 @@
+(ns ^{:author "Andrea Richiardi"
+      :doc "Send + More = Money example + tests, second version." }
+  loco.integer.sendmoremoney2
+  (:refer-clojure :exclude [send]) 
+  (:require [clojure.test :refer :all]
+            [loco.core :refer :all]
+            [loco.constraints :refer :all]
+            [criterium.core :refer [bench]]))
+
+(defn costraint-0-9 [vars]
```

Comment:
Currently it's "costraint" which looks like a typo. I'm okay with the name that's already there if you fix the typo. Or maybe "initialize-digits", since it's initializing a bunch of "digit" (0-9) variables.